apps/src/templates/sectionAssessments/StudentMultipleChoiceOverviewTable.jsx

```diff
@@ -5,6 +5,41 @@ import i18n from '@cdo/locale';
 import wrappedSortable from '../tables/wrapped_sortable';
 import orderBy from 'lodash/orderBy';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import color from "@cdo/apps/util/color";
+
+const TABLE_WIDTH = tableLayoutStyles.table.width;
+const TABLE_COLUMN_WIDTHS = {
+  name: TABLE_WIDTH / 3,
+  numMultipleChoiceCorrect: TABLE_WIDTH / 8,
+  numMultipleChoice: TABLE_WIDTH / 8,
+  percentCorrect: TABLE_WIDTH / 8,
+  timeStamp: TABLE_WIDTH / 5
```

Comment:
all the names here match the property name except for `timeStamp`, can we make this one match what the property says too?